README.md

```diff
@@ -20,4 +37,32 @@ any values in the project's config.py.
 4. To run the development server, execute the following:
 ```bash
 % env/bin/python run.py
-```
\ No newline at end of file
+```
+
+## Static assets development
+
+Javascript and SASS assets are built against Node.js v8.9.3. Usage of
+[nvm](https://github.com/creationix/nvm) is a convenient way to use a specific
+version:
+
+```bash
+nvm use v8.9.3
+```
+
+Node.js dependencies are installed via:
+
+```bash
+npm install
+```
+
+And a development server hosting both the Python and static assets run with:
+
+```bash
+npm run watch
```

Comment:
It is a little confusing to describe how to run the development server above (line 37) and then to have this. I would like to see the running as a separate section. So recommend removing this and have a separate section for running for local development. which will include activating the virtualenv.

Reply:
Good point, will do.